data_structures/src/chain.rs

```diff
@@ -1512,6 +1512,39 @@ impl TransactionsPool {
             && self.re_transactions.is_empty()
     }
 
+    /// Remove all the transactions but keep the allocated memory for reuse.
+    pub fn clear(&mut self) {
+        let TransactionsPool {
```

Comment:
I'm using this syntax instead of `self.vt_transactions.clear();` because this way will stop compiling if we add new fields, so we won't forget to clear them.